ingestion/src/metadata/ingestion/source/database/processor.py

```diff
@@ -18,10 +18,12 @@
 from enum import Enum, auto
 from typing import List, Optional
 
+import spacy
```

Comment:
also sorry I caught it late. But instead of having the processor file at `ingestion/src/metadata/ingestion/source/database/processor.py`, it might make more sense to have it inside `ingestion/processor/pii.py`, wdyt?

Reply:
Ok , that make sense 👍